beacon-chain/rpc/beaconv1/blocks.go

```diff
@@ -70,11 +70,11 @@ func (bs *Server) ListBlockHeaders(ctx context.Context, req *ethpb.BlockHeadersR
 			return nil, status.Errorf(codes.Internal, "Could not retrieve blocks: %v", err)
 		}
 	} else {
-		_, blks, err = bs.BeaconDB.BlocksBySlot(ctx, req.Slot)
+		_, blks, err = bs.BeaconDB.BlocksBySlot(ctx, *req.Slot)
```

Comment:
why are you providing a pointer here

Reply:
Because this is now an optional field, protobuf handles it as a pointer, I'm dereferencing it to use it. I've changed the logic to handle the optional field properly now.